src/main/java/dao/AccountDao.java

```diff
@@ -0,0 +1,132 @@
+package dao;
+
+import card.Card;
+import org.sqlite.SQLiteDataSource;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.Optional;
+
+public class AccountDao implements Dao<Account> {
```

Comment:
The `AccountDao` has too many obligations (create DB, table, connection, and queries to card table).
Can you refactor it?

Reply:
Created a new class `DBCreator`, which is responsible for the db and the account `table` creation.